ignite/cmd/bubblemodel/testnet_multi_node.go

```
@@ -0,0 +1,200 @@
+package cmdmodel
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"syscall"
+
+	tea "github.com/charmbracelet/bubbletea"
+
+	"github.com/ignite/cli/v29/ignite/services/chain"
+)
+
+type NodeStatus int
+
+const (
+	Stopped NodeStatus = iota
+	Running
+)
+
+type MultiNode struct {
+	ctx  context.Context
+	appd string
+	args chain.MultiNodeArgs
+
+	nodeStatuses []NodeStatus
+	pids         []int      // Store the PIDs of the running processes
+	numNodes     int        // Number of nodes
+	logs         [][]string // Store logs for each node
+}
+
+type ToggleNodeMsg struct {
+	nodeIdx int
+}
+
+type UpdateStatusMsg struct {
+	nodeIdx int
+	status  NodeStatus
+}
+
+type UpdateLogsMsg struct{}
+
+func UpdateDeemon() tea.Cmd {
+	return func() tea.Msg {
+		return UpdateLogsMsg{}
+	}
+}
+
+// Initialize the model.
+func NewModel(ctx context.Context, chainname string, args chain.MultiNodeArgs) MultiNode {
+	numNodes, err := strconv.Atoi(args.NumValidator)
+	if err != nil {
+		panic(err)
```

can you return the error and panic only in the top level of the cli?